readme.md

```diff
@@ -1,212 +1,234 @@
+* Perform basic CRUD operations on the database
+* Build and run specific queries
+* Validate schemas
+* Model complex application ideas using embedded data models
+
+## Installation
+
+1. Fork & clone
+
+1. `cp nodemon.sample.json nodemon.json`
+
+1. Create a new Cluster on [MongoDB Atlas](https://www.mongodb.com/cloud/atlas) titled something like `general-purpose` or reuse one you already have.
+
+1. Update the `MONGO_DB_CONNECTION` in your `nodemon.json` file with the full connection string. Make sure you include the password you set up for the database and change the name of the database from `test` to something lke `television_characters_dev`.
+
+1. `npm install`
+
+1. `npm run dev`
+
+Once installation is working, try creating and requesting resources. Note that there is currently no validation set up.
+
+### Instructions & Guiding Questions
+
+- [ ] Take a moment to look through the code that already exists in this repository. Run the code and test out each route, ensuring it does what you expect.
+
+---
+
+- [ ] We will be using both [mongoose](https://mongoosejs.com/docs/guide.html) and [mongodb](https://docs.mongodb.com/manual/) to build this project. Open up each project's documentation.
+
+* **Question:** What is the difference between mongoose and mongodb?
+
+* **Your Answer:**
+
+--- Mongoose is a package used to help store data in mongoDB. MongoDB is the actual database itself. Mongoose helps to format the data in the request.
+
+- [ ] MongoDB uses a number of terms that may be to new to you such as database, collection, and document.
+
+* **Question:** What is the difference between the above terms? [This page](https://docs.mongodb.com/manual/core/databases-and-collections/) may help.
+
+* **Your Answer:**
+
+---A `document` represents a record/collection of data. In mongoDB, it is stored in BSON format, which is similar to JSON. A `collection` is a grouping of documents, usually grouped for a shared or similar purpose, like transactions, or pricing/inventory collections. A `database` contains both documents, invidiually or in collections.
+
+- [ ] Create a new file with the path of `api/models/series.js`. In that file, [define the schema](https://mongoosejs.com/docs/guide.html#definition) for a television series. Include the fields "title", "start_year", and "season_count".
+
+* **Question:** Where does `String` and `Number` come from?
+
+* **Your Answer:**
+
+---String and Number are data types, in the schema they are used to identify the type of value expected for the particular key/value pair.
+
+- [ ] Merge the following into your schema:
+  ```js
+  {
+    _id: Schema.Types.ObjectId
+  }
+  ```
+
+* **Question:** Describe what the above code is adding to your schema.
+
+* **Your Answer:**
+
+---It is used to establish a unique ID for each record.  By default however, MongoDB (Mongoose? This was going to be checked/confirmed) will create a unique ID  even if this statement is not present.

```

Comment:
Mongo sets the ID by default. With both Mongoose & MongoDB, you can disable this.